fury/tests/test_ui.py

```diff
@@ -1146,6 +1146,40 @@ def _on_change():
         show_manager.start()
 
 
+def test_ui_combobox_2d(interactive=False):
+    values = ["An Item" + str(i) for i in range(0, 5)]
+    new_values = ["An Item5", "An Item6"]
+
+    combobox = ui.ComboBox2D(
+        items=values, position=(400, 400), size=(300, 200))
+
+    current_size = (800, 800)
+    show_manager = window.ShowManager(
+        size=current_size, title="ComboBox UI Example")
+    show_manager.scene.add(combobox)
+
+    values.extend(new_values)
+    combobox.append_item(new_values)
+    npt.assert_equal(values, combobox.items)
+
+    values.append("An Item7")
+    combobox.append_item("An Item7")
+    npt.assert_equal(values, combobox.items)
+
+    values.append("An Item8")
+    values.append("An Item9")
+    combobox.append_item("An Item8", "An Item9")
+    npt.assert_equal(values, combobox.items)
+
+    npt.assert_equal(values, combobox.items)
+    npt.assert_equal((60, 60), combobox.drop_button_size)
+    npt.assert_equal([300, 140], combobox.drop_menu_size)
+    npt.assert_equal([300, 200], combobox.size)
```

Comment:
Your tests are nice! thanks! 

However, it seems there is a bug with the scroll. Can you look at it?

<img width="984" alt="Capture d’écran 2020-05-25 à 06 10 51" src="https://user-images.githubusercontent.com/23106443/82777489-8be68700-9e4e-11ea-81d6-9340df03b4e1.png">

Reply:
@skoudoro , can you provide me the code snippet to reproduce this?

Reply:
I just used your test and scrolled down @Nibba2018.

Reply:
Hello @skoudoro , this is happening because the element being stored was a list and not str. We can keep the code base as it is but the user has to add an extra `*` like this instead: `combobox.append_item(*value_list)`. Or we could provide an extra method like `append_item_list` for appending a list object.

Reply:
thank you for fixing it. you can keep `combobox.append_item(*value_list)`

However, I think that you need to add a check on `append_item` function. Something like
```
isinstance(item, list) --> unpack
isinstance(item, str) --> good
isinstance(item, (int,float)) --> str(item)
else --> raise an error
```

Reply:
@skoudoro , I added some checks for `append_item`, let me know what you think about it.

Reply:
Cool and nice 👍 . Looks good

Reply:
Glad you liked it :+1: